packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -177,7 +177,8 @@ function preprocess<T extends z.ZodType>({
         }
 
         if (bookingField.type === "phone") {
-          if (!phoneSchema.safeParse(value).success) {
+          const phoneNumber = value && value.startsWith("+") ? value : `+${value}`
+          if (!phoneSchema.safeParse(phoneNumber).success) {
```

Comment:
@CarinaWolli,

I had the issue in the screenshot below and ended up having to "patch" this file and the zod schema validation.
I don't like my patch and have been investigating the source of the missing "+" from phone number inputs.

A recent commit however changed the react library used for our phone input from "react-phone-number-input" 
to "react-phone-input-2". I'm not sure what to make of this yet but prior to that the "+" prefix was added to the
value of the phone number input.

I'm reaching out to decide what to do about this. I wonder if bookings that require entering phone number are going through (assuming the recent changes have been deployed)

<img width="854" alt="Screenshot 2023-06-01 at 12 25 53 PM" src="https://github.com/calcom/cal.com/assets/119384208/c73c3996-2ffa-4a35-86eb-5dc4c8a38dba">

Reply:
This has been fixed already here: https://github.com/calcom/cal.com/pull/9258

Sorry for that!

Reply:
Thanks.